cli/src/test/java/org/eclipse/hono/cli/client/ReceiverTest.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2019 Contributors to the Eclipse Foundation
+ * Copyright (c) 2016, 2019, 2020 Contributors to the Eclipse Foundation
```

Comment:
same here

Reply:
The format for the period is LOWER BOUNDARY, UPPER BOUNDARY.
In this case, it should be `2016, 2020` ...